examples/flex_plot/flex_common/flex_common/hist.d

```diff
@@ -0,0 +1,132 @@
+module flex_common.hist;
+
```

Comment:
This developed itself into https://github.com/wilzbach/matplotd/blob/master/source/matplotd/hist.d and once matplotd is "a bit" stable I will change the code to use the library instead ;-)